src/style.css

```diff
@@ -0,0 +1,83 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+  font-style: normal;
+}
+
+.form {
+  position: relative;
+  margin: 20px 0;
+}
+
+.search {
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: "Avenir", Arial, sans-serif;
+}
+
+.search--large {
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+}
+
+.search--small {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+}
+
+.search_svg {
+  position: absolute;
+  height: 19px;
+  width: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.smallSearch_svg {
+  position: absolute;
+  height: 11px;
+  width: 11px;
+  left: 13px;
+  top: 15px;
+}
+
+.search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search--large:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-weight: 900;
+  font-style: normal;
+  outline: none;
+  color: #3d4e61;
+}
+
+.search--small:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-style: normal;
+  font-weight: 900;
+  color: #3d4e61;
+  outline: none;
+}
```

Comment:
repeatable styles, so bring rules to common selector `.form__input` and leave different ones at modifiers